models/rne/dirigeants.tsx

```diff
@@ -27,8 +37,7 @@ export const getDirigeantsRNE = async (
     }
 
     try {
```

Comment:
Fallback conditionnel dans le catch :

```suggestion
if (!params.isBot) {
  try {
    return await fallback(siren);
  } catch (eFallback) {
    if (eFallback instanceof HttpNotFound) {
      return APINotRespondingFactory(EAdministration.INPI, 404);
    }
  }
}

return APINotRespondingFactory(EAdministration.INPI, 500);
```

Reply:
Bonne question ! Le retry du fallback pour les bots n'est pas forcément problèmatique ici. La principale cause d'erreur possible ici (autre qu'une 404) est un timeout